data/mods/Magiclysm/worldgen/elf_workshop.json

```
@@ -0,0 +1,77 @@
+[
+  {
+    "type": "palette",
+    "id": "elf_palette",
+    "terrain": { "3": "t_rdoor_c", "1": "t_strconc_wall", ".": "t_metal_floor_olight" },
+    "furniture": {
+      "a": "f_anvil",
+      "t": "f_table",
+      "b": "f_tablesaw",
+      "c": "f_chair",
+      "d": "f_drill_press",
+      "j": "f_jointer",
+      "r": "f_router",
+      "w": "f_workbench",
+      "s": "f_stool"
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "elf_workshop",
+    "object": {
+      "fill_ter": "t_metal_floor",
+      "rows": [
+        "111111111111111111111111",
+        "1a...................ja1",
+        "1tc.....at....d....r.ct3",
+        "1ab..................ct1",
```

```suggestion
        "1bsw.................ct1",
```
I can't see having an anvil next to a table saw.